src/pages/index.tsx

```diff
@@ -94,7 +118,10 @@ const IndexPage: NextPage = () => {
       {appBridgeState?.ready && mounted ? (
         <ClientContent />
       ) : (
-        <p>Install this app in your Dashboard and check extra powers!</p>
+        <div>
+          <p>Install this app in your Dashboard and check extra powers!</p>
+          {mounted && !global.location.href.includes("localhost") && <AddToSaleorForm />}
```

Comment:
app-template can be either opened on localhost or tunnelled. Only with tunneled approach, Saleor can reach the app to install it. Otherwise, dashboard will try to install the app which is not reachable

Reply:
if you are running saleor locally (without the docker), then tunnel is not required for the app to work properly (not all the apps need external webhooks).

But maybe hiding this form will spare a new devs a frustration during development. 🤔 
Lets keep your code

Reply:
Good point. I bet more people will actually connect to cloud, but maybe we will figure out something better in the future